pom.xml

```diff
@@ -35,9 +35,16 @@
   <properties>
     <javaVersion>17</javaVersion>
     <project.build.outputTimestamp>2024-09-29T15:16:00Z</project.build.outputTimestamp>
+
+    <version.slf4j>2.0.16</version.slf4j>
```

Comment:
mvn3 is based on slf4j 1.x

Reply:
mvn3 is stuck forever on plexus-sec-dispatcher 2.0, this is being picked up by mvn4 only